src/Libraries/Microsoft.Extensions.Http.Resilience/Resilience/ResilienceHandlerContext.cs

```diff
@@ -59,7 +59,7 @@ internal ResilienceHandlerContext(AddResiliencePipelineContext<HttpKey> context)
     /// <remarks>
     /// If <paramref name="name"/> is <see langword="null"/> then the global options are returned.
     /// </remarks>
-    public TOptions GetOptions<TOptions>(string name) => _context.GetOptions<TOptions>(name);
+    public TOptions GetOptions<TOptions>(string? name = null) => _context.GetOptions<TOptions>(name);
```

Comment:
Can we also add to the triple slash docs remarks section that if nothing gets passed then null will be used by default?

Reply:
Isn't that implied by default null value?

Reply:
For people looking at the code, yes, but people looking at docs.microsoft.com where this info is surfaced that won't be the case. You can either add it on the remarks, or on the param section instead.

Reply:
Done.